### PR TITLE
Release 13.0.0

### PR DESCRIPTION
**Pull request by jnizet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
 
-All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+All notable changes to this project will be documented in this file. See [commit-and-tag-version](https://github.com/absolute-version/commit-and-tag-version) for commit guidelines.
+
+## [13.0.0](https://github.com/Ninja-Squad/ngx-speculoos/compare/v12.0.1...v13.0.0) (2024-12-06)
+
+
+### ⚠ BREAKING CHANGES
+
+* ngx-speculoos now allows running test in _automatic_ mode. in this mode,
```
Comment (cexbrayat) [inline CHANGELOG.md:10]: nit `In this mode`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+instead of imperatively triggering a change detection, you await the stability
+and thus let Angular decide if a CD is necessary or not.
+See the "Upgrading to v13" section of the README for details.
```
Comment (cexbrayat) [inline CHANGELOG.md:13]: Maybe add a link?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+* ngx-speculoos is now based on Angular 19.x. If you want to use it with an older version of Angular, then stick to a previous version of `ngx-speculoos
```
Comment (cexbrayat) [inline CHANGELOG.md:14]: missing closing backtick

**Review by cexbrayat — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jnizet) [reply CHANGELOG.md:13]: It's a mess because the README is in various places.

**Pull request merged**